### PR TITLE
rename: add page

### PR DESCRIPTION
**Pull request by nikisix**

### CODE REVIEW

**Comment by nikisix** [comment]
Why is it asking for a trailing newline?

File: pages/osx/rename.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 - Replace `from` with `to` in the filenames of the specified files:
 
-`rename 's/{{from}}/{{to}}/' {{*.txt}}`
\ No newline at end of file
+`rename 's/{{from}}/{{to}}/' {{*.txt}}`
```
Comment (nikisix) [inline pages/osx/rename.md:7]: What's the change here?

**Comment by navarroaxel** [comment]
@nikisix The UNIX-standard in an empty line at the EOF for every text file (programming languages, logs, etc).
so every file should end with `\n`

File: pages/osx/rename.md (hunk 1 above)
Comment (navarroaxel) [reply pages/osx/rename.md:7]: Yep, just an enter.

**Comment by nikisix** [comment]
I see. Just checked my `~/.git/config` and found:

```
[core]
	precomposeunicode = true
```

That the problem perhaps?

**Comment by nikisix** [comment]
Anyways, thanks for the awesome tool. I'll try and add more stuff in the future now that I know the guidelines and all.

**Review by navarroaxel — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good to me.

**Pull request merged**